docs/ManualDeploymentGuideScalarDLAuditorOnAWS.md

```diff
@@ -64,23 +64,23 @@ so you need to add peering for internal communication between the Auditor, Ledge
     * Peering between Ledger and Client.
     * Peering between Auditor and Client.
 * Update route tables for VPC peering connection based on the [AWS official guide](https://docs.aws.amazon.com/vpc/latest/peering/vpc-peering-routing.html).
-* Add new Inbound and Outbound rule to restrict unwanted access to Ledger and Auditor.
-    * Add new Inbound rules to the Ledger Network ACLs to allow the Ledger Envoy LoadBalancer (e.g., 50051 and 50052 by default) access from the Auditor and Client.
+* Add new Inbound and Outbound rules to restrict unwanted access to Ledger and Auditor.
+    * Add new Inbound rules to Ledger Network ACLs to allow ephemeral ports (1024–65535) to access Ledger Envoy LoadBalancer (e.g., 50051 and 50052 by default) from the Auditor and Client.
         * You must set high priority to this rule.
```

Comment:
The above point is about adding `rules` to LedgerNetwork ACL. But this one says to set high priority to this rule.
If multiple rules have to be set, it will be confusing as to which rule should have high priority.

Reply:
Thank you for your review comments.
Document updated with `rules` instead of `rule` 31dedd5

Reply:
High priority or higher priority than something? What do you mean `high priority here?
Also, indenting is unnecessarily deep.